### PR TITLE
Live page support

### PR DESCRIPTION
**Pull request by brmighell**
It works! Check out the [live page](https://brmighell.github.io/rcv-entry/) for proof! Once this pull request is merged I'll swap the page source back to `main`.

### CODE REVIEW

File: data-table/table.js [hunk 1]
```diff
@@ -305,6 +306,47 @@ function createRowEntryBox(config) {
...
+ * as well as a button that sends input text to addSingleColumn().
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function createColumnInputAndBtn(config) {
+    let entryBoxDiv = document.getElementById(config.tableIds.entryBoxDivId);
+    // Creates the field for text input
+    let input = document.createElement("INPUT");
+    input.type = 'text';
+    input.placeholder = "Enter " + config.datumConfig.names;
```
Comment (artoonie) [inline data-table/table.js:320]: `datumConfig` is for the specific value type - think % chance of precipitation, or numVotes.

You want `config.columnsName` here, I think?

File: data-table/table.js [hunk 2]
```diff
@@ -305,6 +306,47 @@ function createRowEntryBox(config) {
...
+     * TODO: Button will need to accept a number of columns from the user then pass that number to addMultipleColumns
+     */
+    let addColumnBtn = document.createElement("button");
+    addColumnBtn.innerHTML = "Add column to right";
+    addColumnBtn.onclick = function () {
+        let value = input.value.trim();
+        if (value !== '') {
+            addSingleColumn(config, config.numColumns, value);
+        } else {
+            addSingleColumn(config, config.numColumns, config.datumConfig.names[0])
```
Comment (artoonie) [inline data-table/table.js:343]: Similarly, I believe you want `config.columnsName` here.

It looks like we are assuming here that both the rows and the columns are modifiable. @khashshafiei can you add a story to make sure we handle this configuration option? Specifically, whether rows or columns have modifiable titles should be configurable.

For example, in the RCVis use case: rows (candidate names) is configurable, but columns (rounds) is not.

**Review by artoonie — COMMENTED** [review]
1. Looks like `addSingleColumn` ("Add column to right") doesn't work
2. To keep git history clean, consider using `git commit --amend` to amend the previous commit instead of pushing a new one

File: data-table/table.js (hunk 1 above)
Comment (brmighell) [reply data-table/table.js:320]: Yup, totally right. Made the change.

**Review by fotakis08 — APPROVED** [review]
Looks like its coming together! Great work.

**Pull request merged**